wcfsetup/install/files/lib/data/media/MediaAction.class.php

```diff
@@ -182,6 +188,11 @@ protected function getMediaData($media) {
 				'id' => $media->userID,
 				'title' => $media->username
 			]) : '',
+			// TODO: find better solution
```

Comment:
TODO. This is really ugly.

Reply:
That is why I also explicitly added to the message of this pull request that I am open for better solutions :)

May-be we should add a method to UserProfile which relies on an actual template with `{user object=$userProfile}`.

Reply:
I'm not happy about it, but adding bogus extra methods is only moving the issue somewhere else. Moving it (into an actual template) is only masquerading the issue, not solving it.

For the lack of a truly better solution: Keep it.